src/xcompact3d.f90

```diff
@@ -30,8 +32,14 @@ program xcompact3d
 
   telapsed = 0
   tmin = telapsed
-  ndt = 1
-
+  ndt = 0
+
+  ioutput = 5
```

Comment:
Took me a bit of scrolling to understand what that variable was about, maybe a name like `io_period`?